scipy/integrate/_quadrature.py

```diff
@@ -531,7 +532,7 @@ def simps(y, x=None, dx=1.0, axis=-1, even=None):
     return simpson(y, x=x, dx=dx, axis=axis, even=even)
 
 
-def simpson(y, x=None, dx=1.0, axis=-1, even=None):
+def simpson(y, x=None, dx=1.0, axis=-1, even=_NoValue):
```

Comment:
This is how things would look like after switching to keyword-only args.
```suggestion
def simpson(y, *, x=None, dx=1.0, axis=-1):
```
In the meantime, we should probably vendor the [helper](https://github.com/scikit-learn/scikit-learn/blob/1.2.2/sklearn/utils/validation.py#L42) that scikit-learn came up with to do the deprecation of positional use.

Reply:
I am +1000 there!